openjdk/build.xml

```diff
@@ -77,7 +77,7 @@
 			<arg line="-d ${jtregTar}.tar.gz" />
 		</exec>
 		<if>
-			<contains string="${SPEC}" substring="zos" />
+			<equals arg="${BASE_OS}" arg2="zos" />
```

Comment:
I didn't see `SPEC` set anywhere in the file, so I'm assuming this is launched from either `JenkinsfileBase` or `openjdk_tests`.